client/service/src/builder.rs

```diff
@@ -1328,6 +1328,7 @@ ServiceBuilder<
 			let subscriber = sc_tracing::ProfilingSubscriber::new(
 				config.tracing_receiver, tracing_targets
 			);
+			sp_tracing::set_wasm_tracing(config.tracing_enable_wasm);
```

Comment:
Why are all these internal details not handled by the `ProfilingSubscriber` internally?